parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoParquetReader.java

```diff
@@ -56,7 +61,25 @@ public ProtoParquetReader(Path file) throws IOException {
    */
   @Deprecated
   @SuppressWarnings("unchecked")
-  public ProtoParquetReader(Path file, UnboundRecordFilter recordFilter) throws IOException {
+  public ProtoParquetReader(Path file, UnboundRecordFilter recordFilter)
+      throws IOException {
```

Comment:
Untouched code should not be formatted. (BTW, the max line width is 120 characters anyway.)